server/src/test/java/io/crate/types/NumericTypeTest.java

```diff
@@ -66,9 +66,9 @@ public void test_implicit_cast_text_to_unscaled_numeric() {
         assertThat(NumericType.INSTANCE.implicitCast("-12839")).isEqualTo(BigDecimal.valueOf(-12839));
         assertThat(NumericType.INSTANCE.implicitCast("+2147483647111")).isEqualTo(BigDecimal.valueOf(2147483647111L));
         assertThat(NumericType.INSTANCE.implicitCast("+214748364711119475")).isEqualTo(new BigDecimal("214748364711119475"));
+        assertThat(NumericType.INSTANCE.implicitCast("21474836.4711119475")).isEqualTo(new BigDecimal("21474836.4711119475"));
```

Comment:
What will this cover additionally?

Reply:
only a case where the text represents a floating point number, not just integrals, can remove if you think it is superfluous.

Reply:
Hm it such seems to test the behaviour `BigDecimal(string)` then. Not needed, but no strong opinion.